recipes/medcoupling/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "9.10.0" %}
+
+package:
+  name: medcoupling
+  version: '{{ version }}'
+
+source:
+  - fn: 'medcoupling_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/medcoupling.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    sha256: 02087a4e59ffcdab6fc68460ffd86b03a80b5c0faa695b0bfb2142f1d0d3e5ea
+    patches:
+      - swig_big_file_compile_flag.patch
+  - fn: 'config_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/configuration.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    folder: deps/config
+    sha256: d58f6749514ce49b00590a96d6dc9090a76a07c13690e2453403643afb40346c
+
+
+build:
+  skip: True  # [osx]
+  detect_binary_files_with_prefix: true
+  # string: py{{ CONDA_PY }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [mpi != "nompi"]
+  # string: py{{ CONDA_PY }}_nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}           # [mpi == "nompi"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make                                      # [not win]
+    - ninja                                     # [win]
+    - cmake
+    - swig >=4.0.2
+
+  host:
+    - python
+    - numpy
+    - boost
+    - zlib
+    - hdf5
+    - libxml2
+    - libmed
+    - petsc                                     # [mpi != 'nompi']
```

Comment:
it seems that I add metis enabled also in my pr

Reply:
I see. As far as I know, METIS when used as a dependency for Code Aster, EDF uses a patched version of METIS that enables support for both 64 and 32 bit integers. Last I checked the conda-forge version of METIS did not have these changes.

As of now we are looking to support 64 bit integers on linux and 32 bit integers on windows.
Given that some of us wish to use `med` and `medcoupling` with Code Aster (see https://github.com/conda-forge/code-aster-feedstock where we currently use static compilation and embed the dependencies directly instead of relying on other conda-forge packages), I suggest we skip adding METIS as a dependency until we are certain that conda-forge versions of METIS contains the patches.

Reply:
For reference, here is the metis-5.1.0 patch file

[metis-5.1.0_aster4.patch](https://github.com/conda-forge/staged-recipes/files/14601523/metis-5.1.0_aster4.patch)